app/main.js

```diff
@@ -66,29 +66,29 @@ if (process.env.NODE_ENV === 'development') {
   });
 }
 
-// Listen for a 'resin-update-lock' to either enable, disable or check
+// Listen for a 'update-lock' to either enable, disable or check
 // the update lock from the renderer process (i.e. the app)
-if (electronConfig.ELECTRON_RESIN_UPDATE_LOCK) {
+if (electronConfig.ELECTRON_BALENA_UPDATE_LOCK) {
   const lockFile = require('lockfile');
-  electron.ipcMain.on('resin-update-lock', (event, command) => {
+  electron.ipcMain.on('update-lock', (event, command) => {
```

Comment:
I've just noticed that the example code in `README.md` is using e.g. `ipcRenderer.once('balena-update-lock', (event, error) => {`
Does it need to be updated to instead use the `'update-lock'` name that is being used in `main.js` ?